src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -732,6 +731,50 @@ public void OptionSetChecks()
             Assert.True(checkResult.IsSuccess);
         }
 
+        [Theory]
+        [InlineData("OptionSetInfo(OptionSet.option_1)", "option_1")]
+        [InlineData("OptionSetInfo(OptionSet.Option1)", "option_1")]
+        [InlineData("OptionSetInfo(Option1)", "option_1")]
+        [InlineData("OptionSetInfo(If(1<0, Option1))", null)]
+        public async void OptionSetInfoTests(string expression, string expected)

```

Comment:
>OptionSetInfoTests [](http://example.com/codeflow?start=26&length=18)

Can we include test cases with Text() .

Helpful to see OptionSetInfo and Text() together to better understand their relationship.

Reply:
updated